x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/api.ts

```diff
@@ -160,7 +160,7 @@ export const deleteRules = async ({ ids }: DeleteRulesProps): Promise<Array<Rule
   const response = await KibanaServices.get().http.fetch<Rule[]>(
     `${DETECTION_ENGINE_RULES_URL}/_bulk_delete`,
     {
-      method: 'PUT',
+      method: 'DELETE',
```

Comment:
Looks like I had forgot to update this verb as part of https://github.com/elastic/kibana/pull/54521 -- thanks for this cleanup! 🙂